File: week6/lecture/phonebook.py
from cs50 import get_string

people = {
    "Brian": "+1-617-495-1000",
    "David": "+1-496-588-1260",
    "Brian": "+1-736-685-2387"
}

name = get_string("Name: ")
if name in people:
    number = people[name]
    print(f"Number: {number}")

File: week3/lecture/phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

typedef struct
{
    string name;
    string number;
}
person;

const int NUMBER = 10;

int main(void)
{
    person people[NUMBER];

    for (int i = 0; i < NUMBER; i++)
    {
        people[i].name = get_string("Whats the name? ");
        people[i].number = get_string("Whats their number? ");
    }

    for (int i = 0; i < NUMBER; i++)
    {
        if (strcmp(people[i].name, "David") == 0)
        {
            printf("Found %s\n", people[i].number);
            return 0;
        }
    }
    printf("Not found\n");
    return 1;
}